packages/flutter/lib/src/material/slider.dart

```diff
@@ -520,6 +527,7 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
       overlayEntry!.remove();
       overlayEntry = null;
     }
+    _focusNode?.dispose();
```

Comment:
Probably not too much of a deal. I just noticed we never dispose focus node in didUpdateWidget in other widget as well if widget.focusNode go from null to non-null. I imagine we should, WDYT @gspencergoog.

Reply:
Ah nvm, I think that will break Focus widget if the focusnode is currently attached....